server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraCacheConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package org.apache.james.blob.cassandra.cache;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+import javax.inject.Inject;
+
+import org.apache.commons.configuration2.Configuration;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public class CassandraCacheConfiguration {
+
+    private final String TIME_OUT_PROPERTY = "";
+    private final String MAX_SIZE_PROPERTY = "";
+
+    private final long DEFAULT_TIME_OUT = 50;
+    private final int DEFAULT_MAX_SIZE = 8;
+
+    final Duration TIME_OUT;
+    final int MAX_SIZE;
+
+    @Inject
+    public CassandraCacheConfiguration(Configuration propertiesConfiguration) {
```

Comment:
Let's provide a clean pojo and it's builder, let's keep guice injections for later...